web-components/jest/svgTransform.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+module.exports = {
+  process() {
+    return "module.exports = '<svg></svg>';";
+  },
+  getCacheKey() {
```

Comment:
in the jest tests this will be what gets returned when you request an svg. previously it was using the imgTransform which leads to parsing errors